.golangci.yml

```diff
@@ -10,6 +10,11 @@ issues:
   max-same-issues: 0
   exclude:
     - "G114: Use of net/http serve function that has no support for setting timeouts"
+  exclude-rules:
+    # ref: https://github.com/dominikh/go-tools/issues/1365
+    - path: client/pchainclient.go
+      linters:
+        - unused
```

Comment:
@bilgehansahin what is this for?

Reply:
The linter started to complain about the type aliases inside pchainclient.go (platformvmClient, indexerClient and infoClient) being unused despite them being embedded in the main client. It appears to be a known issue with the linter. This disables that check. #201 lint fails for the same reason.